plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -270,7 +270,11 @@ function BookInfoManager:loadSettings(db_conn)
         local keys = res[1]
         local values = res[2]
         for i, key in ipairs(keys) do
-            self.settings[key] = values[i]
+            local value = values[i]
+            if value then
+                value = tonumber(value) or value -- convert cdata<int64_t> to lua number 
```

Comment:
Technically it's going to be a string, because the SQL table has a `TEXT` type, but eh, that's the spirit ;p.